libs/application/templates/passport-annulment/src/forms/sections/info.ts

```diff
@@ -43,6 +43,7 @@ export const info = buildMultiField({
       width: 'half',
       largeButtons: false,
       space: 'containerGutter',
+      defaultValue: 'lost',
```

Comment:
Would it be safer to create an enum for these?

Something like

```typescript
enum PassportStatus {
  Lost = 'lost',
  Stolen = 'stolen',
}
```

Just to prevent typos and make it clearer what is being used elsewhere when actions are taken f.x.